pandas/core/generic.py

```diff
@@ -3042,7 +3042,7 @@ def to_sql(
         For MySQL, a callable to update columns ``b`` and ``c`` if there's a conflict
         on a primary key.
 
-        >>> from sqlalchemy.dialects.mysql import insert
+        >>> from sqlalchemy.dialects.mysql import insert  # doctest: +SKIP
```

Comment:
Nice changes, just one comment. The error seems to suggest that `insert` is already imported before but never used. Can you have a look, and remove the previous import if that's really the case.

Reply:
It's actually used in L3036:
```
...     stmt = insert(table.table).values(data).on_conflict_do_nothing(index_elements=["a"])
```
I am not quite sure why it shows this error, and thus I think we could just  `# doctest: +SKIP` this line